haystack/utils/deepsetcloud.py

```diff
@@ -951,6 +1079,23 @@ def get_evaluation_set_client(
         client = DeepsetCloudClient(api_key=api_key, api_endpoint=api_endpoint)
         return EvaluationSetClient(client=client, workspace=workspace, evaluation_set=evaluation_set)
 
+    @classmethod
+    def get_eval_run_client(
+        cls, api_key: Optional[str] = None, api_endpoint: Optional[str] = None, workspace: str = "default"
+    ) -> EvaluationRunClient:
+        """
+        Creates a client to manage eval runs on deepset Cloud.
+
+        :param api_key: Secret value of the API key.
+                        If not specified, will be read from DEEPSET_CLOUD_API_KEY environment variable.
+        :param api_endpoint: The URL of the deepset Cloud API.
```

Comment:
I think currently it needs to be set to: https://api.cloud.deepset.ai/api/v1 so can we add: The endpoint should be: https://api.cloud.deepset.ai/api/v1.